astrality/utils.py

```diff
@@ -140,7 +140,7 @@ def resolve_targets(
 
     else:
         for file in content.glob('**/*'):
-            if file.is_dir():
+            if file.is_dir() and not file.is_symlink():
```

Comment:
Not necessarily true. We might want to symlink to symlinks

    modules/vim/:    .vimrc    <----    .ideavimrc
                       ↑                    ↑ 
    $HOME       :    .vimrc             .ideavimrc